shopify-client/src/views/ShopDetails.vue

```diff
@@ -85,7 +87,6 @@
                 this.$router.push(`/app/shops/${this.id}/update`);
             }
         },
-        props: ["id"],
```

Comment:
I want to keep the path variable.

Reply:
https://medium.com/@fullsour/when-should-you-use-path-variable-and-query-parameter-a346790e8a6d. Basically if you want to identify a resource you want to use path variable.

Reply:
reverted